public/coffee/ide/editor/directives/aceEditor/auto-complete/AutoCompleteManager.coffee

```diff
@@ -121,6 +121,8 @@ define [
 			end = change.end
 			range = new Range(end.row, 0, end.row, end.column)
 			lineUpToCursor = @editor.getSession().getTextRange(range)
+			if lineUpToCursor.slice(0, 1) == '%'
```

Comment:
This will only catch comments like:

```
% This is a full line comment
```

Not comments like:

```
$x^2$ % This is  partial line comment.
```

Could just make it a regex looking for any % (Without a preceding \) before the cursor.